src/server/ftpserver/options.rs

```diff
@@ -16,6 +16,7 @@ pub(crate) const DEFAULT_PASSIVE_HOST: PassiveHost = PassiveHost::FromConnection
 pub(crate) const DEFAULT_PASSIVE_PORTS: Range<u16> = 49152..65535;
 pub(crate) const DEFAULT_FTPS_REQUIRE: FtpsRequired = FtpsRequired::None;
 pub(crate) const DEFAULT_FTPS_TRUST_STORE: &str = "./trusted.pem";
+pub(crate) const DEFAULT_SITEMD5: SiteMd5 = SiteMd5::Accounts;
```

Comment:
Might want to do 

```
impl Default for SiteMd5 {
    fn default() -> SiteMd5 {
        SiteMd5:: Accounts
    }
}

```

instead, up to you.

Reply:
yeah better